### PR TITLE
Updated interface link in document

### PR DESCRIPTION
**Pull request by gelanivishal**
## This PR is a:

- [ ] New topic
- [X] Content update
- [X] Content fix or rewrite
- [X] Bug fix or improvement

## Summary
Updated interface link on Magento document.
When this pull request is merged, it will...

## Additional information

List all affected URLs 
- https://devdocs.magento.com/guides/v2.3/config-guide/config/config-create.html
- https://devdocs.magento.com/guides/v2.2/config-guide/config/config-create.html
- https://devdocs.magento.com/guides/v2.1/config-guide/config/config-create.html

### CODE REVIEW

**Comment by magento-cicd2** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.0/config-guide/config/config-create.md [hunk 1]
```diff
@@ -37,12 +37,12 @@ For example, to introduce an {% glossarytooltip edb42858-1ff8-41f9-80a6-edf0d86d
...
 
-*  `$fileResolver`. Implements `\Magento\Framework\Config\FileResolverInterface`. This parameter lists the files containing the configurations of your custom type.
-*  `$converter`. Implements `\Magento\Framework\Config\ConverterInterface`. This parameter converts the XML into the internal array representation of the configurations.
-*  `$schemaLocator`. Implements `\Magento\Framework\Config/SchemaLocatorInterface`. This parameter provides the full path to file(s) containing schema(s) for validation of the individual and merged configuration files.</p>
-*  `$validationState`. Implements `\Magento\Framework\Config\ValidationStateInterface`. This parameter defines whether a configuration file should be validated. 
+*  `$fileResolver`. Implements [\Magento\Framework\Config\FileResolverInterface]({{ site.mage2300url }}lib/internal/Magento/Framework/Config/FileResolverInterface.php). This parameter lists the files containing the configurations of your custom type.
```
Comment (keharper) [inline guides/v2.0/config-guide/config/config-create.md]: The varisbles in lines 42, 43, and 44 need to be changed to mage2000url

Comment (gelanivishal) [reply guides/v2.0/config-guide/config/config-create.md]: @keharper Done.

**Comment by keharper** [comment]
running tests

**Comment by keharper** [comment]
I'm sorry, @gelanivishal , I can't merge this. There are still symbolic link issues. I would create a new branch and run the following commands from the devdocs directory


1. git rm guides/v2.3/config-guide/config/config-create.md
2. git rm guides/v2.2/config-guide/config/config-create.md
3. cd guides/v2.1/config-guide/config
4. cp config-create.md ../../../v2.3/config-guide/config/config-create.md
5. cp config-create.md ../../../v2.2/config-guide/config/config-create.md

Then make your documentation changes as before.

These steps will get rid of the symbolic links and allow us to publish.

**Comment by gelanivishal** [comment]
@keharper I have performed as the same thing but don't know. Can you please create your side then I'll update it.

**Comment by keharper** [comment]
@gelanivishal  I've updated the repo so that the files are no longer symlinked. Pull those changes in and hopefully we can get this PR closed

**Comment by gelanivishal** [comment]
@keharper I have changed links on new files. Please check it.

**Comment by keharper** [comment]
running tests

**Comment by keharper** [comment]
For some reason the symlinks are still a problem. Please open a new PR, but don't update the 2.0 version, since that's an unsupported branch. Because I got rid of the symlinks in #3470, there shouldn't be any more problems.

**Pull request closed without merging**